thetanuts/thetanuts_test.py

```diff
@@ -181,9 +183,9 @@ def get_env(variable: str) -> str:
     )["errors"]
     is False
 ):
-    print("Bid validated")
+    print(" -> Bid validated")
 else:
-    print("Bid validation failed!")
+    print(" -> Bid validation failed!")
```

Comment:
Can't comment the exact line (193) because it is not modified and github does not allow me to add comments to unchanged lines.

verify_allowance is duplicated, I think we can remove the [first call from lines 193 to 199](https://github.com/tradeparadigm/sdks/blob/d5011f54b1466e7df9e7fe31ddbd6b817ef73253/thetanuts/thetanuts_test.py#L193-L199)